disc/cogs/map_pack.py

```diff
@@ -17,9 +17,11 @@ async def pack(self, ctx: commands.Context, *osu_urls: str):
         map_ids = [MapIdFactory.from_url(url) for url in set(osu_urls)]
         maps = await Maps.from_map_ids(map_ids)
         map_pack = maps.zip()
-
         if map_pack.stat().st_size > 25e6:
-            await ctx.reply(f"Map pack is too large to send: {map_pack.stat().st_size} bytes")
+            await ctx.reply("The file(s) you have provided us with is larger than the upload limit for Discord, "
+                            "please wait as we upload it to dropbox")
+            url = maps.upload("/mappapakka/") # TODO: Give these file uploads a nice name?
```

Comment:
i don't think there is any easy way for a nice name unless we think of an order independent hash for the files inside the zip

uuid is good because it is guaranteed unique so no 2 files will ever conflict